.github/workflows/azops.yml

```diff
@@ -3,88 +3,90 @@ name: AzOps
 on:
   repository_dispatch:
     types:
-      - "activity-logs"
+      - "Azure Activity Logs"
+      - "GitHub CLI"
+      - "PowerShell"
+      - "Bash"
   pull_request:
     branches:
-      - main
+      - "main"
     paths:
       - "azops/**"
   pull_request_review:
     branches:
-      - main
-    types: [submitted]
+      - "main"
+    types:
+      - "submitted"
     paths:
       - "azops/**"
 
+env:
+  AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+  AZOPS_STATE: "azops"
+  AZOPS_DEFAULT_DEPLOYMENT_REGION: "northeurope"
+  AZOPS_ENROLLMENT_ACCOUNT: ""
+  AZOPS_OFFER_TYPE: "MS-AZR-0017P"
+  AZOPS_IGNORE_CONTEXT_CHECK: "0"
+  AZOPS_INVALIDATE_CACHE: "1"
+  AZOPS_SKIP_RESOURCE_GROUPS: "1"
+  AZOPS_SKIP_POLICY: "0"
+  AZOPS_THROTTLE_LIMIT: "10"
+  GITHUB_USERNAME: "AzOps"
+  GITHUB_EMAIL: "noreply@azure.com"
+  GITHUB_PULL_REQUEST: "Azure Change Notification"
+  GITHUB_CONTEXT: ${{ toJson(github) }}
+  GITHUB_ISSUE: ${{ github.event.pull_request._links.issue.href }}
+  GITHUB_COMMENTS: ${{ github.event.pull_request._links.comments.href }}
+  GITHUB_BASE_REF: ${{ github.event.pull_request.base.ref }}
+  GITHUB_HEAD_REF: ${{ github.event.pull_request.head.ref }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  VERBOSE: "0"
+  DEBUG: "0"
+
 jobs:
+
   push:
-    if: ${{ github.event_name == 'pull_request' && github.head_ref != 'system' }}
+    if: github.head_ref != 'system' && github.event_name == 'pull_request'
     name: Push
     runs-on: ubuntu-latest
     steps:
       - name: Actions checkout
         uses: actions/checkout@v2
       - name: GitHub context
-        env:
-          GITHUB_CONTEXT: ${{ toJson(github) }}
         run: echo "$GITHUB_CONTEXT"
       - name: Actions azops
         uses: Azure/AzOps@main
```

Comment:
Is the intent to always use the original `Azure/AzOps` repository, or should we provide the option to set this via a GitHub secret (with a default of `Azure/AzOps@main` when no secret value provided) so users can override this with their own copy / specific release version if wanted?

Just thinking about this in the context of our discussion yesterday on the team call when we were discussing pinning to specific releases.

Reply:
This went through my mind last night; I've not included yet, need to explore options, let's aim to add this in a subsequent pull request.